capture_service/device_mgr.cc

```diff
@@ -512,6 +515,14 @@ absl::Status DeviceManager::RunReplayApk(const std::string &capture_path,
 {
     LOGD("RunReplayApk(): starting\n");
 
+    bool pinned_clock = true;
```

Comment:
i am wondering if you could create a helper class and put this code block to the constructor, and the next code block to the destructor so that we can avoid duplicating the code in both RunReplayApk and RunProfilingOnReplay

Reply:
sounds good